recipes/spdlog/all/conanfile.py

```diff
@@ -152,8 +147,7 @@ def package_info(self):
         if self.options.header_only and self.settings.os in ("iOS", "tvOS", "watchOS"):
             self.cpp_info.components["libspdlog"].defines.append("SPDLOG_NO_TLS")
 
-        if Version(conan_version).major < 2:
```

Comment:
OOC: why was this remove? I'd really love to not see warnings in Conan 2.0 and was actually going to suggest adding something like this into recipes.

Reply:
Now that 2.0 runs are submitted as part of the report from CCI, I think that noise is good in this case, as you say, we want to catch as many bugs as possible before GA!